docs: Add more readable argo CLI installation guide. Fixes #11750 (#11751)
argoproj/argo-workflows: docs/quick-start.md

## Install the Argo Workflows CLI

> Next, for easy use of Argo Workflows, it is recommended to install and use the Argo CLI.  
> You can check an overview of the installation and usage on the [Argo CLI Walk Through page](https://argoproj.github.io/argo-workflows/walk-through/argo-cli/).

Reviewer: ```suggestion
You can more easily interact with Argo Workflows with the [Argo CLI](https://argoproj.github.io/argo-workflows/walk-through/argo-cli/).
```
- [active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice) and [more direct](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)

Author: Thanks for noticing about #9412 and the style guide. I read your recommendations and agree. Removing the duplications and just adding simple install tip on Argo CLI page would be better.
